packages/core/package.json

```diff
@@ -3,5 +3,10 @@
   "version": "0.0.1",
   "dependencies": {},
   "type": "commonjs",
-  "main": "./index.cjs"
+  "main": "./index.cjs",
+  "peerDependencies": {
+    "chalk": "5.3.0",
+    "@quality-metrics/models": "latest",
+    "@quality-metrics/utils": "latest"
```

Comment:
Again, we need to align on how we reference versions between our packages. I'm not sure `latest` and `*` are very future proof.